conf/init/data_migration.py

```diff
@@ -0,0 +1,10 @@
+from app import app
+from active_migration import ActiveDataMigration
+
+if not app.config.get("SETUP_COMPLETE", False):
+    print "head"
+else:
+    if ActiveDataMigration is not None:
+        print ActiveDataMigration.alembic_migration_revision
```

Comment:
For Python3 preparation, it might be useful to change these `print` statements to `print(...)`.

Reply:
I'd rather not change them in this PR, since they aren't being added but rather moved